### ISSUE TITLE
Replace BLACK, WHITE, and polish functions

### ISSUE THREAD

**Issue by amalyam** [seas-computing/mark-one#216]
## Description

Within the `MarkOneTheme`, we need to replace the `BLACK` and `WHITE` constants with true white (`#FFFFFF`) and true black (`#000000`), as they currently use colors that are slightly off from those. We also need to replace any colors that use the `polish` library functions `lighten` and `darken` with hex codes. These changes will make our theme more straightforward, so it's easier to determine accessible color combinations  moving forward.

The hex code equivalents for the lighten and darken functions are approximately as follows (worth checking the styleguide as these updates are made, to see if the hex code is actually equivalent to the previous function output):
color name | function | hex code
-- | -- | -- 
background.darker | darken(0.75, WHITE) |  #3f3f3f
background.base.medium | darken(0.175, WHITE) |  #d1d1d1
text.light | lighten(0.9, BLACK) | #fefefe
font.note | lighten(0.35, BLACK) | #717171
font.title | lighten(0.35, BLACK) | #717171
border.light | lighten(0.5, BLACK) | #979797
border.heavy | lighten(0.3, BLACK) | #646464

## Acceptance Criteria

- [ ] the BLACK and WHITE constants use `#000000` and `#FFFFFF` respectively instead of the current colors
- [ ] the 7 `polish` functions are replaced with hex codes that approximately match the current colors

## Related Issues
[dev-team-notes #29](https://github.huit.harvard.edu/SEAS/dev-team-notes/issues/29)
